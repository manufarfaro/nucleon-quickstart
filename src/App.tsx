import React from 'react';
import { createUseStyles } from 'react-jss';

const useStyles = createUseStyles({
  root: {
    width: '100%',
    height: '100%',
    backgroundColor: '#475f8c',
    color: '#ffffff',
    fontSize: '23px',
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center'
  }
});

function App() {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <span>Hello React! :)</span>
    </div>
  );
}

export default App;
