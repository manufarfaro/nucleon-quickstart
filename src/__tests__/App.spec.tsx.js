import React from 'react'
import { render, cleanup } from '@testing-library/react'
import App from '../App';

describe('App component', () => {
  afterEach(() => {
    cleanup();
  })

  describe('when is called', () => {
    it('should render correctly', () => {
      const { asFragment } = render(<App />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
