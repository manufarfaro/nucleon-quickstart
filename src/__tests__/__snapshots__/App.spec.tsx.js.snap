// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component when is called should render correctly 1`] = `
<DocumentFragment>
  <div
    class="root-0-2-1"
  >
    <span>
      Hello World
    </span>
  </div>
</DocumentFragment>
`;
